plugins/paytovote/paytovote.go

```diff
@@ -0,0 +1,183 @@
+package paytovote
+
+import (
+	"fmt"
+
+	abci "github.com/tendermint/abci/types"
+	"github.com/tendermint/basecoin/types"
+	"github.com/tendermint/go-wire"
+)
+
+const (
+	TypeByteCreateIssue byte = 0x00
+	TypeByteVoteFor     byte = 0x01
+	TypeByteVoteAgainst byte = 0x02
+	TypeByteVoteSpoiled byte = 0x03
+)
+
+type P2VPluginState struct {
+	TotalCost    types.Coins
+	Issue        string
+	votesFor     int
+	votesAgainst int
+	votesSpoiled int
+}
+
+type P2VTx struct {
+	Valid            bool
+	Issue            string      //Issue being voted for
+	ActionTypeByte   byte        //How is the vote being cast
+	Cost2Vote        types.Coins //Cost to vote
+	Cost2CreateIssue types.Coins //Cost to create a new issue
```

Comment:
Oh maybe this is because Counter includes Cost in the Tx.  I added that just to test things, so I can see how it's misleading.  We should remove Cost from Counter too, make it a fee, and set the cost upon counter.New(name, cost).